docs/fsharp/style-guide/formatting.md

```diff
@@ -1140,6 +1140,68 @@ Option.traverse(
 )
 ```
 
+Formatting generic type arguments and constraints
+-------------------------------------------------
```

Comment:
This isn't valid, you'll need to do use `#` characters to designate a new section.

Reply:
Fixed. My tooling uses that style; I didn't check the source for the other headings.